docker-compose.deploy.yml

```diff
@@ -27,3 +29,14 @@ services:
       - FLASK_SECRET_KEY
       - CSSIGNIN_CONFIG=/appdata/cssignin.yaml
       - LANG=en_US.UTF-8
+
+  postgres:
+    container_name: chopshop_db
+    image: postgres:16r1
```

Comment:
Should this be `r1` or `rc1`?

Reply:
They've finally pushed  a postgres:16 tag so using that.